package/src/lsif.ts

```diff
@@ -111,7 +115,13 @@ export const mkIsLSIFAvailable = () => {
             if (!response.ok) {
                 return false
             }
-            return await response.json()
+            const available: boolean = await response.json()
+            if (available) {
+                console.log(
+                    `LSIF data is available for repository=${repository} commit=${commit} file=${file}`
```

Comment:
This is still a bit ambiguous - it's available, but does that mean it's used?
-------

Would recommend logging an object `{ repository, commit, file }` because Chrome will highlight that and make it expandable